runtime/src/slots.rs

```diff
@@ -242,10 +242,11 @@ decl_module! {
 		/// This can only happen when there isn't already an auction in progress and may only be
 		/// called by the root origin. Accepts the `duration` of this auction and the
 		/// `lease_period_index` of the initial lease period of the four that are to be auctioned.
-		fn new_auction(
+		fn new_auction(origin,
```

Comment:
Please put this parameter onto a new line.

Reply:
I was just following the pattern in the other functions. I should make an update to them all?